octoprint_mrbeam/static/js/lib/hopscotch.min.js

```diff
@@ -1318,19 +1318,19 @@
                             '\n  </div>\n  <div class="hopscotch-actions">\n    '),
                         g.showPrev &&
                             (e +=
-                                '<button class="hopscotch-nav-button prev hopscotch-prev">' +
+                                '<button class="btn prev hopscotch-prev">' +
```

Comment:
This is a minified file of the library. We cannot modify it. Please find another way to do that.